summit/2018.html

```diff
@@ -36,12 +36,31 @@ <h2 class="location">Location</h2>
         <a href="http://oficina.al" target="_blank" rel="noopener"><img class="sponsor" src="/images/summit/oficina.svg" width="250px" height="auto" alt="Ofiçina"/></a><br />
         <a href="https://osm.org/go/xexSY_f5h--?m=" target="_blank" rel="noopener">Rruga Asim Vokshi<br />Tirana, Albania</p><img class="map" src="/images/summit/map.jpg" width="600px" height="347px" alt="Map"/></a>
       </div>
+
+      <div class="row" style="margin-bottom: 30px; margin-top: 30px;">
+      <div class="col-md-10 col-md-offset-1">
+        <img class="sponsor" src="/images/summit/agenda.svg" width="65px" height="auto"
+            alt="Sponsor"/>
+        <h2 class="text">Schedule</h2>
+        <p class="data"><strong>2 November</strong><br/>
+        4 PM - 8 PM: Reception and Planning for the Weekend Topics</p>
+        <p class="data"><strong>3 November</strong><br/>
+        10 AM-  1 PM : Open Source Design Meta Discussions<br />
+        1 PM - 2 PM: Lunch<br />
+        2 PM - 8 PM: Unconference Sessions</p>
+        <p class="data"><strong>4 November</strong><br />
+        10 AM - 1 PM: Working Sessions HandsOn<br />
+        1 PM - 2 PM Lunch<br />
+        2 PM - 6 PM Working Sessions & Open Source Design in 2019</p>
+        </div>
```

Comment:
Would have liked a left-align for the agenda (in order to easily scan the time), but the whole page is centered, so it's ok.

Reply:
Yeah, it's a problem because the whole page is centered